### ISSUE TITLE
Percent sign after number won't be spaced

### ISSUE THREAD

**Issue by Ynjxsjmh** [coldnew/pangu-spacing#31]
Say I have some text like
```
迈克尔写了 99% 的歌词，但是他觉得没必要说出来。
```
I think there should have a space after percent sign % and before Chinese character 的 like the example I give above.

I also find [pangu.js](https://github.com/vinta/pangu.js)'s description in your README saying:
> This plugin (support
> Chrome and Firefox) is what you need when surfing the
> internet, it will add whitespace between Chinese,
> half-width English, digits and symbols automatically.

However what `pangu-spacing` does is
```
迈克尔写了 99%的歌词，但是他觉得没必要说出来。
```

**Comment by coldnew** [comment]
Yes, currently the `pangu-spacing` is not a clone of [pangu.js](https://github.com/vinta/pangu.js), so their functionality are different.

I have plans to backport my Clojure/Script port of [pangu.clj](https://github.com/coldnew/pangu.clj) to `pangu.el`, which is in `develop` branch or `pangu-spacing`, but the work is in slow progress...
